TechnicoBackend/Services/PropertyService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TechnicoBackend.Models;
+using TechnicoBackend.DTOs;
+using TechnicoBackend.Repositories;
+
+namespace TechnicoBackend.Services
+{
+    public class PropertyService
```

Comment:
Consider using an interface IPropertyService from which the current class would be inherited.